aws/extensions/computetask_awswin_ospatching/extension.ftl

```diff
@@ -0,0 +1,74 @@
+[#ftl]
+
+[@addExtension
+    id="computetask_awswin_ospatching"
+    aliases=[
+        "_computetask_awswin_ospatching"
+    ]
+    description=[
+        "Creates a cron based yum update for security patching"
+    ]
+    supportedTypes=[
+        EC2_COMPONENT_TYPE,
+        ECS_COMPONENT_TYPE,
+        COMPUTECLUSTER_COMPONENT_TYPE,
+        BASTION_COMPONENT_TYPE
+    ]
+    scopes=[
+        COMPUTETASK_EXTENSION_SCOPE
+    ]
+/]
+
+[#macro shared_extension_computetask_awswin_ospatching_deployment_computetask occurrence ]
+
+    [#local OSPatching = _context.InstanceOSPatching ]
+    [#local schedule = OSPatching.Schedule ]
+    [#local securityOnly = OSPatching.SecurityOnly ]
+
+    [#local updateCommand = "yum clean all && yum -y update"]
+
+<!-- Windows instances are patched by new AMIs
+    [#if OSPatching.Enabled ]
+ -->
```

Comment:
The awslinux instances are as well. The AMIs are released in monthly cycles I think. so could miss some updates that have been released in between. the AMI cycle

Reply:
Not sure this is the same for Windows as patches are released on "Patch Tuesday" (ie monthly) and the AMIs are updated with 5 working days thereafter. I'm not aware of people using automated patching on Windows servers, but I'll have a dig through Windows Update

Reply:
Lets leave it for now and instead add a warning or fatal message that if OSPatching is Enabled on the solution configuration let the user know at template generation time that the standard OS patching rules apply. 

If someone wants to enable patching they can add their own extension

Reply:
Cool, done

Reply:
I'm getting this warning even though I've specified OSPatching/Enabled:false in masterdata for the _awswin DeploymentProfile. Any advice on what I've done wrong?

Reply:
Is this still happening or was it resolved?

Reply:
Still happening

Reply:
Should now be fixed in b211b66 
It looks like the Environment Patching configuration was overriding the instance. The instance should be preferred as its the most specific